recipes-openxt/tss/tpm2-tss/0001-disable-code-coverage.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/Makefile.am b/Makefile.am
+index d78d23f..2ee64e0 100644
+--- a/Makefile.am
++++ b/Makefile.am
+@@ -40,10 +40,6 @@ CLEANFILES =
+ MOSTLYCLEANFILES =
+ noinst_PROGRAMS =
+ 
+-### Add ax_* rules ###
+-# ax_code_coverage
+-@CODE_COVERAGE_RULES@
+-
```

Comment:
Why are we dropping code coverage check?

Reply:
I couldn't get tpm2-tss to build without removing this. OE didn't know what to do with the `@CODE_COVERAGE_RULES@` macro; it kept it as-is in the Makefile. Kept seeing the following:
```
Log data follows:
| DEBUG: SITE files ['endian-little', 'bit-32', 'ix86-common', 'common-linux', 'common-glibc', 'i686-linux', 'common']
| DEBUG: Executing shell function do_compile
| NOTE: make -j 4
| ERROR: oe_runmake failed
| Makefile:13115: *** missing separator (did you mean TAB instead of 8 spaces?).  Stop.
```
If you have another solution to this I will gladly use it.

Reply:
Looks like ax_code_coverage.m4 comes from autoconf-archive, so there should be a DEPENDS on that.  However, that file isn't packages in meta-oe since they expect gnome-common to provide it.  Looks like we already depend on gnome-common, so maybe just try adding that to DEPENDS?